.github/workflows/run-tests.yaml

```diff
@@ -61,7 +52,7 @@ jobs:
         env:
           FORCE_COLOR: 1
           PYTEST_CI_ARGS: --cov-report=xml --cov-report=html --junitxml=test_artifacts/test_report.xml --color=yes
-        run: tox ${{ matrix.python-version == '3.6' && '--skip-missing-interpreters=true' || '' }}
+        run: tox ${{ matrix.python-version == '3.12' && '--skip-missing-interpreters=true' || '' }}
```

Comment:
That was ONLY a 3.6-required thing I did and _could've been dropped_ together with it AFAIK.

Did you formally verify this as "needed"?

Reply:
I've updated the command to avoid this altogether.